src/Phansible/Roles/Blackfire.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Phansible\Roles;
+
+use Phansible\BaseRole;
+
+class Blackfire extends BaseRole
+{
+    protected $name = 'BlackFire';
+    protected $slug = 'blackfire';
+    protected $role = 'blackfire';
+
+    public function getInitialValues()
+    {
+        return [
+          'install' => 0,
+          'server_id' => '',
+          'server_token' => '',
+        ];
+    }
+
+    protected function installRole($requestVars)
+    {
+        return parent::installRole($requestVars) && $this->blackfireWillBeInstalled($requestVars);
+    }
+
+    private function blackfireWillBeInstalled($requestVars)
```

Comment:
You don't need to do this since this is already done in `installRole` in the `BaseRole`